### PR TITLE
Video importer

### PR DESCRIPTION
**Pull request by dougdroper**

### CODE REVIEW

**Comment by tomas-stefano** [comment]
:+1:

File: lib/cms/hippo_importer/base.rb [hunk 1]
```diff
@@ -33,7 +33,7 @@ def parent
       end
 
       def docs
-        @cached_docs ||= @docs.empty? ? Comfy::Cms::Page.all.map(&:slug) : @docs
+        @cached_docs ||= @docs.empty? ? Comfy::Cms::Page.all.map(&:slug) : @docs.map(&:strip)
```
Comment (benlovell) [inline lib/cms/hippo_importer/base.rb]: Can you not `pluck(:slug)` here?

Comment (dougdroper) [inline lib/cms/hippo_importer/base.rb]: Yip, all done :+1:

**Comment by tomas-stefano** [comment]
:+1:

**Pull request merged**